DorisProject/src/app/pages/categorias/categorias.component.ts

```diff
@@ -51,28 +54,69 @@ addCategory(){
   }
 
   search(event: any) {
-    const SearchList =[];
+    const SearchList = [];
     if (event.target.value == "")
-        this.categorias= this.categoriasMaster;
-      else
-      {
-          this.categoriasMaster.forEach(category =>{
-            console.log(category)
-          if(category.description.toLowerCase().includes(event.target.value.toLowerCase()) )
-            {
-              SearchList.push(category);
-              return;
-            }
-            if(category.name.toLowerCase().includes(event.target.value.toLowerCase()) )
-            {
-              SearchList.push(category);
-              return;
-            }
-          });
-          this.categorias=SearchList;
+      this.categorias = this.categoriasMaster;
+    else {
+      this.categoriasMaster.forEach(category => {
+        console.log(category)
+        if (category.description.toLowerCase().includes(event.target.value.toLowerCase())) {
+          SearchList.push(category);
+          return;
+        }
+        if (category.name.toLowerCase().includes(event.target.value.toLowerCase())) {
```

Comment:
Codacy has a fix for the issue: [Replace `category.name.toLowerCase().includes(event.target.value.toLowerCase())` with `⏎··········category.name.toLowerCase().includes(event.target.value.toLowerCase())⏎········`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9277326)
```suggestion
        if (
```